flow-typed/npm/babel-cli_vx.x.x.js

```diff
@@ -0,0 +1,108 @@
+// flow-typed signature: 8615cf6c39596ff7ba27c4a0377fe299
```

Comment:
This was entirely auto-generated and only really affects linting so I didn't think it was worth a PR on it's own.